tests/org.eclipse.swt.tests.win32/JUnit Tests/org/eclipse/swt/graphics/WidgetWin32Tests.java

```diff
@@ -0,0 +1,59 @@
+package org.eclipse.swt.graphics;
+
+import static org.junit.Assert.assertEquals;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.internal.DPITestUtil;
+import org.eclipse.swt.internal.DPIUtil;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.swt.widgets.Event;
+import org.eclipse.swt.widgets.Shell;
+import org.eclipse.swt.widgets.WidgetTestUtil;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class WidgetWin32Tests {
```

Comment:
Why is this class placed in the `graphics` instead of the `widgets` package? The `WidgetTestUtil` also seems to be a symptom of this potential misplacement.

Reply:
Just wanted to have all these tests for multi zoom level in one package. But if its not a problem then I can move them to separate packages.